packages/components/webpack.config.js

```diff
@@ -1,24 +1,26 @@
 const path = require("path");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
```

Comment:
I can't figure out why webpack should be aware of css at all. Won't `tailwindcss -i ... -o ...` invocation be enough?